README.md

```diff
@@ -1,3 +1,67 @@
+# The Ambassador Agent
+
+An optional component included by default with [Telepresence](https://github.com/telepresenceio/telepresence) and [Ambassador Edge Stack](https://github.com/emissary-ingress/emissary).
```

Comment:
Add the agent as the subject of the sentence:
_"The Ambassador Agent is an optional component included with [Telepresence](https://github.com/telepresenceio/telepresence) and [Ambassador Edge Stack](https://github.com/emissary-ingress/emissary)."_